packages/backend-test-utils/src/next/services/MockAuditorService.ts

```diff
@@ -0,0 +1,195 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type {
+  AuditorEvent,
+  AuditorEventOptions,
+} from '@backstage/backend-defaults/auditor';
+import type {
+  AuditorCreateEvent,
+  AuditorService,
+  AuthService,
+  BackstageCredentials,
+  HttpAuthService,
+  PluginMetadataService,
+} from '@backstage/backend-plugin-api';
+import { ForwardedError } from '@backstage/errors';
+import type { JsonObject } from '@backstage/types';
+import type { Request } from 'express';
+import type { mockServices } from './mockServices';
+
+export class MockAuditorService implements AuditorService {
```

Comment:
With the default auditor service simply forwarding to the logger service I don't think we need a concrete mock implementation and can use the default implementation instead